src/services.js

```diff
@@ -634,3 +634,21 @@ export class Services {
     return /** @type {!./service/xhr-impl.Xhr} */ (getService(window, 'xhr'));
   }
 }
+
+/**
+ * @param {!Element} element
+ * @param {function()} mutator
+ */
+export function resourcesMutateElement(element, mutator) {
```

Comment:
this discourages caching a reference to `Services.resourcesForDoc(element)` when such cases is needed. I don't think we should add it.